gax-java/gax/src/test/java/com/google/api/gax/retrying/ScheduledRetryingExecutorTest.java

```diff
@@ -249,6 +249,7 @@ public void testCancelGetAttempt() throws Exception {
     }
   }
 
+  // This test also verifies that the tracer has cancelled the attempt
```

Comment:
I think the comment on the verification should be good enough.

Reply:
Sounds good. Will remove this line.